cfme/scripting/quickstart.py

```diff
@@ -226,6 +226,9 @@ def link_config_files(src, dest):
                 if os.readlink(target) != source:
                     print("WARNING:", target, "does not point to", source)
                     print("         please verify this is intended")
+            elif os.path.isfile(target):
+                print('ERROR: You have', element, 'copied into your conf/ folder. Remove it.')
```

Comment:
@RonnyPfannschmidt , I suspect this is normal situation when someone keeps env.yaml file instead of symlink. Are you sure script should exit instead of just throwing warning  ?